source/manual/2nd-line.html.md

```diff
@@ -7,50 +7,115 @@ section: 2nd line
+
+Note that frontend developers are not expected to be on-call rota.
+
+> Technical 2nd Line takes priority over the work you do in your usual team.
+
+Shifts start at 9:30 and end at 17:30. [You can check the Technical 2nd Line rota to find out when your shift is][2ndline-rota].
+
+[2ndline-rota]: https://docs.google.com/spreadsheets/d/1OTVm_k6MDdCFN1EFzrKXWu4iIPI7uR9mssI8AMwn7lU/edit#gid=1297388378.
+
+You are required to attend a daily morning standup with your paired 2nd Line partner and the 2nd Line team. There’s a short retrospective at the handover meeting at the end of your shift.
+
+If you have meetings to attend then attend them. Please let the delivery manager and the team know when you’ll be away for long periods, and be respectful of the amount of work your colleagues may have to pick up while you’re away. If there are lots of alerts, or there's a live incident, or an urgent Zendesk ticket, you’ll need to prioritise Technical 2nd Line above your meetings.
+
+## Shift swaps, working patterns and sickness
+
+2nd Line takes priority over the work you do in your usual team. If you need to swap your shift, it’s your responsibility to ensure that adequate cover is in place.
+
+- If you need cover for a day or two, arrange a swap for those days with another developer. Please ensure delivery managers are aware of this.
+
+- If you need a whole shift swap, arrange this with another developer from your team.
+
+For either of the above, let the Technical 2nd Line delivery manager know. Please update the schedule on [PagerDuty][].
+
+If you cannot make your shift because you’re ill, message the delivery manager and #govuk-2ndline-tech Slack channel.
+
+If your working patterns are not compatible with a 9.30am-5.30pm shift, let the Technical 2nd Line team know so they can find extra support.
+
+If you do not work a 5-day week, please talk to your delivery manager to arrange cover with another developer on your team.
 
 ## Monitoring
 
 We have a [Technical 2nd Line dashboard][] showing a high level overview of the state of the GOV.UK environments. You can also [install our Chrome extension][] if you want a permanently visible overview. You will need to be on the [VPN](/manual/vpn.html) if accessing from home.
 
 ### Icinga
 
-We use Icinga to monitor our platform and alert us when things go wrong. Many alerts have corresponding  documentation in these developer docs, detailing how to respond.
+We use Icinga to monitor our platform across our environments ([CI/Integration][], [Staging][], [Production][]) and alert us when things go wrong. Many alerts have corresponding documentation in our [GOV.UK developer docs][], detailing how to respond. [Read more about Icinga][].
+
+If you see critical alerts that aren't easily solved, start a thread in the #govuk-2ndline-tech Slack channel.
+
+NB: Technical 2nd Line should investigate these alerts when there is downtime; you do not necessarily have to fix them.
```

Comment:
I think we can lose this line - we should be aiming to resolve or acknowledge all alerts.